bot/client.js

```diff
@@ -57,4 +57,10 @@ client.on('message', (message) => {
   }
 });
 
+client.on('guildMemberAdd', (member) => {
+  member.sendMessage('Welcome to the channel!');
```

Comment:
Why is this feature in your DB scheme branch? This was never merged into the dev branch, so it should not be here unless you have not followed the DWA branch model.